### PR TITLE
handle duplication of output on kndp create cmd

### PR DESCRIPTION
**Pull request by ca7alindev**

### CODE REVIEW

File: cmd/kndp/environment/create.go [hunk 1]
```diff
@@ -61,12 +61,18 @@ func (c *createCmd) Run(ctx context.Context, p pterm.TextPrinter) error {
 
 	stderrScanner := bufio.NewScanner(stderr)
 	for stderrScanner.Scan() {
-		fmt.Println(stderrScanner.Text(), "1")
+		line := stderrScanner.Text()
+		if !strings.Contains(line, " • ") {
+			fmt.Println(line)
```
Comment (web7evgheni) [inline cmd/kndp/environment/create.go:66]: @ca7alindev Please try to configure kind to not display "fancy" messages, as i understood is possible by special environment variables, which says to kind, that terminal do not supports such kind of messages.

**Review by web7evgheni — CHANGES_REQUESTED** [review]

**Comment by web7evgheni** [comment]
@ca7alindev Check this function, there possible to switch Kind output into different mode:
https://github.com/kubernetes-sigs/kind/blob/main/pkg/internal/env/term.go#L51

**Comment by web7evgheni** [comment]
@ca7alindev How you did that?

**Comment by ca7alindev** [comment]
@web7evgheni by setting some env variables as suggested in https://github.com/kubernetes-sigs/kind/blob/main/pkg/internal/env/term.go#L51

**Comment by web7evgheni** [comment]
@ca7alindev How did you set envs?

**Comment by ca7alindev** [comment]
@web7evgheni in terminal with export, like "export TERM=dumb", the output was changed in simpler one, but sill with double lines

**Comment by web7evgheni** [comment]
@ca7alindev Do you think Go give your TTY environment variables to exec commands by default? :)

**Review by web7evgheni — APPROVED** [review]

**Pull request merged**